sqlagg/base.py

```diff
@@ -73,6 +73,10 @@ def execute(self, metadata, connection, filter_values):
         query = self._build_query(metadata)
         return connection.execute(query, **filter_values).fetchall()
 
+    def count(self, metadata, connection, filter_values):
+        query = self._build_query(metadata).count()
```

Comment:
> I don't see this kind of usage of count in the SQLAlchemy docs. 

```count```: http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.count

> Does this still work if the query has a limit or group by applied?

Most of the time I expect this to be used with ```group_by```.
I suppose this works with ```limit``` but I deliberately don't want it used to count the number of rows in a paginated query, so I added an ```assert```.